pkg/storage/common/common.go

```diff
@@ -95,7 +95,17 @@ func ValidateManifest(imgStore storageTypes.ImageStore, repo, reference, mediaTy
 			return "", zerr.ErrBadManifest
 		}
 
-		if manifest.Config.MediaType == ispec.MediaTypeImageConfig {
+		// validate blobs only for known media types
+		if manifest.Config.MediaType == ispec.MediaTypeImageConfig ||
+			manifest.Config.MediaType == ispec.MediaTypeEmptyJSON {
```

Comment:
I thought there is no blob uploaded for ispec.MediaTypeEmptyJSON, as the data is in the descriptor itself.
What's the point of having images with ispec.MediaTypeEmptyJSON if they still have a `{}` config blob separately?

Reply:
That is still an actual blob of size=2. Hence nothing special in terms of uploads and existence.

Reply:
If that was correct, why go though all the overhead of including it in the spec? It doesn't add any benefit, compared to me defining my own {} config blob media type and it would be ingested just the same according to the spec.

Reply:
Convention still matters.